pkg/container-disk/container-disk_test.go

```diff
@@ -414,23 +415,24 @@ var _ = Describe("ContainerDisk", func() {
 				Expect(err).ToNot(HaveOccurred())
 				Expect(res).To(Equal(expected))
 			},
-				Entry("docker", "docker://sha256:4gjffGJlg4"),
-				Entry("dontainerd", "sha256:4gjffGJlg4"),
-				Entry("cri-o", "myregistry/myimage@sha256:4gjffGJlg4"),
+				Entry("docker", "docker://sha256:4gjffGJlg421e150ae84b0973863609494aad70f0a80eaeb64bddd8d92465812"),
+				Entry("dontainerd", "sha256:4gjffGJlg421e150ae84b0973863609494aad70f0a80eaeb64bddd8d92465812"),
+				Entry("cri-o", "myregistry/myimage@sha256:4gjffGJlg421e150ae84b0973863609494aad70f0a80eaeb64bddd8d92465812"),
+				Entry("cri-o >= 1.28", "myregistry/myimage@sha256:4gjffGJlg421e150ae84b0973863609494aad70f0a80eaeb64bddd8d92465812"),
```

Comment:
This `Entry` should be:
```
Entry("cri-o >= 1.28", "4gjffGJlg421e150ae84b0973863609494aad70f0a80eaeb64bddd8d92465812")
```
otherwise it is exaclty the same of cri-o < 1.28

Reply:
done